pkgs/tools/networking/lldpd/default.nix

```diff
@@ -10,6 +10,15 @@ stdenv.mkDerivation rec {
     sha256 = "0lgiappbjm95r1m0xyxb6gzz4izcjixknbzq3s7pbqbsmhm642s5";
   };
 
+  remove_backported_patch = version != "1.0.1";
```

Comment:
Thats a clever pattern, but I guess it could lead to confusion. Won't the patch fail to apply on future versions anyways?

Reply:
Since you've backported a patch that is already included upsteram (right?) the patch will fail to apply after the next update, since the relevant code is already changed.

Thanks for your effort in fixing this :)

Reply:
OK, good. I'll remove the extra cruft. Indeed this is from upstream and it confirmed to be in the next release.

https://github.com/vincentbernat/lldpd/commit/b0ca585f627aad476604a11bd356b842c664fb84
https://github.com/vincentbernat/lldpd/commit/90a50860ebdcdeb5b7dc85790b18bed23c97ec32

Reply:
Great :) Don't forget to ping me once you've removed it as github doesn't always send notifications on code changes.